isrtask.rst

```diff
@@ -178,6 +179,6 @@ Algorithm details
 
 -------------
   
-  [Extra reference: Section 4 of LSST DATA CHALLENGE HANDBOOK (2011), and http://hsca.ipmu.jp/public/index.html ]
+  [Extra reference: Section 4 of LSST DATA CHALLENGE HANDBOOK (2011) [https://project.lsst.org/sciencewiki/images/DC_Handbook_v1.1.pdf] , and http://hsca.ipmu.jp/public/index.html ]
```

Comment:
This link is actually where i got it, and is still live, so in some way, we may want to include this ref, or the appropriate DMTN later.